docs/user/troubleshooting/boot-rescue/system-not-working-after-update.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: My system doesn't start after an update
+summary: Troubleshooting the boot process for Solus
+---
+
+import BootAndChroot from "./snippets/_mount_partitions.mdx";
+import UnmountAll from "./snippets/_unmount_partitions.mdx";
+import WhatToDoNext from "./snippets/_contact_team.mdx";
+
+## Before your start
+
+For this procedure, you need:
+
+- [A Solus Live USB](../../quick-start/installation/#creating-bootable-media)
+- Your disk encryption password, if your system is encrypted
+- A stable internet connection
+
+## Procedure
+
+1. Shut down your system.
+1. Boot from a Solus live USB.
+1.  Mount the partitions of your Solus system.
+
+    <details>
+      <BootAndChroot />
+    </details>
+
+1. Check the chroot environment has internet connection.
+    <details>
+    	1. Connect your computer to the internet.
+    	2. Run `ping -c 4 google.com` in the chroot environment.
+    
+           If the chroot environment has internet connection, the system displays the following:
+    
+           ```bash
+           PING google.com (142.250.196.206) 56(84) bytes of data.
+           64 bytes from nctsaa-ac-in-f14.1e100.net (142.250.196.206): icmp_seq=1 ttl=115 time=4.26 ms
+           64 bytes from nctsaa-ac-in-f14.1e100.net (142.250.196.206): icmp_seq=2 ttl=115 time=6.04 ms
+           64 bytes from nctsaa-ac-in-f14.1e100.net (142.250.196.206): icmp_seq=3 ttl=115 time=6.34 ms
+           64 bytes from nctsaa-ac-in-f14.1e100.net (142.250.196.206): icmp_seq=4 ttl=115 time=5.81 ms
+                  --- google.com ping statistics ---
+           4 packets transmitted, 4 received, 0% packet loss, time 3004ms
+           rtt min/avg/max/mdev = 4.261/5.613/6.344/0.803 ms
+           ```
+        3. If the chroot environment *does not* have an internet connection:
+        	1. Exit the chroot environment:
+    
+        	```bash
+        	exit
+        	```
+    
+        	2. Run the following commands:
+    
+        	```bash
+        	cp /run/systemd/resolve/stub-resolv.conf /target/run/systemd/resolve/
+        	cd /target/etc
+            ln -s ../run/systemd/resolve/stub-resolv.conf /target/etc/resolv.conf
+        	```
```

Comment:
It looks like these code blocks have the same indentation issue. I didn't see them the first time.